repo2docker/buildpacks/r.py

```diff
@@ -19,8 +19,22 @@ class RBuildPack(PythonBuildPack):
        date snapshot of https://mran.microsoft.com/timemachine
        from which libraries are to be installed.
 
-    2. An optional `install.R` file that will be executed at build time,
-       and can be used for installing packages from both MRAN and GitHub.
+    2. A `DESCRIPTION` file signaling an R package
+
+    3. A Stencila document (*.jats.xml) with R code chunks (i.e. language="r")
+
+    If there is no `runtime.txt`, then the MRAN snapshot is set to latest
+    date that is guaranteed to exist across timezones.
+
+    R packages are installed if specified either
+
+    - in a file `install.R`, that will be executed at build time,
+      and can be used for installing packages from both MRAN and GitHub
+
+    - as dependencies in a `DESCRIPTION` file
+
+    - are needed by a specific tool, for example the package `stencila` is
+      installed and configured if a Stencila document is given.
```

Comment:
I think this bit is about installing user specified packages so we should only list the ways by which a user can specify "please install X and Y" instead of listing the situations in which extra packages will be installed from a developer point of view.

The suggestion on L29 is connected to this.